Loop/Window Management/Window.swift

```diff
@@ -225,7 +225,7 @@ class Window {
         }
     }
 
-    /// MacOS doesn't provide us a way to find the minimum size of a window from the accessibility API.
+    /// macOS doesn't provide us a way to find the minimum size of a window from the accessibility API.
```

Comment:
This is the start of a sentence, so "macOS" should be reverted to "MacOS"

Reply:
Updated.